tests/helper_functions.py

```diff
@@ -15,22 +15,17 @@
 import sys
 import yaml
 
-import pandas as pd
 import pytest
 
 import mlflow
-import mlflow.pyfunc.scoring_server as pyfunc_scoring_server
-import mlflow.pyfunc
 from mlflow.tracking.artifact_utils import _download_artifact_from_uri
 from mlflow.utils.file_utils import read_yaml, write_yaml
-from mlflow.utils.model_utils import _get_flavor_configuration, FLAVOR_CONFIG_CODE
 from mlflow.utils.environment import (
     _get_pip_deps,
     _CONDA_ENV_FILE_NAME,
     _REQUIREMENTS_FILE_NAME,
     _CONSTRAINTS_FILE_NAME,
 )
-from mlflow.utils.requirements_utils import _get_installed_version
```

Comment:
Tuck imports with dependencies that are not skinny-friendly (e.g. pandas) inside the helper functions where they're actually used